app/src/main/java/com/kickstarter/ui/activities/ActivityFeedActivity.kt

```diff
@@ -0,0 +1,145 @@
+package com.kickstarter.ui.activities
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.kickstarter.R
+import com.kickstarter.databinding.ActivityFeedLayoutBinding
+import com.kickstarter.libs.*
+import com.kickstarter.libs.qualifiers.RequiresActivityViewModel
+import com.kickstarter.libs.rx.transformers.Transformers
+import com.kickstarter.libs.utils.ApplicationUtils
+import com.kickstarter.libs.utils.ObjectUtils
+import com.kickstarter.models.*
+import com.kickstarter.ui.IntentKey
+import com.kickstarter.ui.adapters.ActivityFeedAdapter
+import com.kickstarter.ui.data.LoginReason
+import com.kickstarter.viewmodels.ActivityFeedViewModel
+
+@RequiresActivityViewModel(ActivityFeedViewModel.ViewModel::class)
+class ActivityFeedActivity : BaseActivity<ActivityFeedViewModel.ViewModel>() {
+    private var adapter: ActivityFeedAdapter? = null
+    private var currentUser: CurrentUserType? = null
+    private var recyclerViewPaginator: RecyclerViewPaginator? = null
+    private var swipeRefresher: SwipeRefresher? = null
+    private lateinit var binding: ActivityFeedLayoutBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityFeedLayoutBinding.inflate(layoutInflater)
+        val view: View = binding.root
+        setContentView(view)
+
+        currentUser = environment().currentUser()
+        adapter = ActivityFeedAdapter(viewModel.inputs)
+        binding.recyclerView.adapter = adapter
+        binding.recyclerView.layoutManager = LinearLayoutManager(this@ActivityFeedActivity)
+
```

Comment:
nit: Add whitespace to separate these codeblocks and subscriptions to improve readability